### DISCUSSION TITLE
Would you please help me to install?

### DISCUSSION THREAD

**Discussion by TVARecordings** [halitsever/random-memes#34] [General]
Hello!

So, i've downloaded a large file to run in VCS, using Java with Nodes, which runs as a whatsapp-bot.

I'd like to integrate your feature into the bot, but i clearly have no clue how to do so.

I understand a little bit about the script. 

The commands are handled in 'MessageHandler'
For example: 
case 'makequote':
        if (arguments.join(' ').split('@').length < 2) return await client.reply(from, '_⚠️ Example of using the command: !makequote <Name>@<Sentence>_', id);
        const getMakequote = _function.makequote(arguments);
        await client.sendImage(from, getMakequote, sender.id, '', id);
break;

I do know that i have to download your node into the 'node_modules' folder, but i do not know how to link the commandlines.

**Comment by TVARecordings** [comment]
Guess not

**Reply by halitsever** [reply]
Hey, sorry for my late reply, if you are still looking for how to integrate random-memes to your bot, you can check this example:
https://github.com/halitsever/random-memes/blob/master/examples/Telegram-Bot-Random-Meme-Generator-Example/Telegram-Bot-Random-Memes.js

This example for telegram, but you can integrate this code for whatsapp.
